CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ All notable changes to this project will be documented in this file.
 - Added `ProductVersion` to status ([#81]).
 - Added `Condition` to status ([#81]).
 
+### Changed
+
+- `kube-rs`: `0.58` → `0.60` ([#83]).
+- `k8s-openapi` `0.12` → `0.13` and features: `v1_21` → `v1_22` ([#83]).
+- `operator-rs` `0.2.1` → `0.2.2` ([#83]).
```

Comment:
It's a bit too late now I guess but I only now realize: The upgrade from 0.58 to 0.59 was a BREAKING change, that should have meant we do a version bump from 0.2 to 0.3 in operator-rs. Next time.